Directory.Build.targets

```diff
@@ -1,301 +1,296 @@
-                </Reference>
-                <Reference Include="UnityEngine.UnityWebRequestWWWModule">
-                    <HintPath>$(GameManagedDir)\UnityEngine.UnityWebRequestWWWModule.dll</HintPath>
-                    <Private>$(TestLibrary)</Private>
-                </Reference>
-                <Reference Include="UnityEngine.VehiclesModule">
-                    <HintPath>$(GameManagedDir)\UnityEngine.VehiclesModule.dll</HintPath>
-                    <Private>$(TestLibrary)</Private>
-                </Reference>
-                <Reference Include="UnityEngine.VFXModule">
-                    <HintPath>$(GameManagedDir)\UnityEngine.VFXModule.dll</HintPath>
-                    <Private>$(TestLibrary)</Private>
-                </Reference>
-                <Reference Include="UnityEngine.VideoModule">
-                    <HintPath>$(GameManagedDir)\UnityEngine.VideoModule.dll</HintPath>
-                    <Private>$(TestLibrary)</Private>
-                </Reference>
-                <Reference Include="UnityEngine.VRModule">
-                    <HintPath>$(GameManagedDir)\UnityEngine.VRModule.dll</HintPath>
-                    <Private>$(TestLibrary)</Private>
-                </Reference>
-                <Reference Include="UnityEngine.WindModule">
-                    <HintPath>$(GameManagedDir)\UnityEngine.WindModule.dll</HintPath>
-                    <Private>$(TestLibrary)</Private>
-                </Reference>
-                <Reference Include="UnityEngine.XRModule">
-                    <HintPath>$(GameManagedDir)\UnityEngine.XRModule.dll</HintPath>
-                    <Private>$(TestLibrary)</Private>
-                </Reference>
-            </ItemGroup>
-        </When>
-    </Choose>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <Target Name="FindGameAndIncludeReferences" BeforeTargets="ResolveAssemblyReferences" Condition="'$(UnityModLibrary)' == 'true' or '$(TestLibrary)' == 'true'">
+        <DiscoverGame GameName="Subnautica" IntermediateOutputPath="$(IntermediateOutputPath)">
+            <Output TaskParameter="GamePath" PropertyName="GameDir" />
+        </DiscoverGame>
+        <Error Condition="'$(GameDir)' == ''" Text="Failed to find the game 'Subnautica' on your machine" />
+        <PropertyGroup>
+            <GameDir>$(GameDir)\</GameDir>
+            <GameManagedDir>$(GameDir)Subnautica_Data\Managed\</GameManagedDir>
+            <GameVersionFile>$(GameDir)Subnautica_Data\StreamingAssets\SNUnmanagedData\plastic_status.ignore</GameVersionFile>
+            <GameLegacyVersion>68598</GameLegacyVersion>
+        </PropertyGroup>
+        <ReadLinesFromFile File="$(GameVersionFile)">
+            <Output TaskParameter="Lines" ItemName="GameVersion" />
+        </ReadLinesFromFile>
+        <Error Condition="@(GameVersion) &lt;= $(GameLegacyVersion)" Text="Subnautica must not be on legacy branch. Expected version &gt;$(GameLegacyVersion) but was @(GameVersion)" />
```

Comment:
Shouldn't the "Expected Version" here be the latest version of Subnautica instead of Legacy? Or is the ">" symbol meant to say that its supposed to be greater than the legacy version?